### ISSUE TITLE
Insta360 support

### ISSUE THREAD

**Issue by amedrz** [cgsecurity/testdisk#163]
Hi, I'm wondering if it's possible to add support for `.insp` and `.lrv` files? Here's the links from PhotoRec online checker:

- [`.insp` file](https://www.cgsecurity.org/photorec/scan.php?file=4a83bdf8c651d4631b80570a59364f2f4b046df28ef94c0eb6cde9359b231bb7)
- [`.lrv` file](https://www.cgsecurity.org/photorec/scan.php?file=ffaa54c399cff99e99a54f0b2855a65eb7d197614377adc7b96683cbe0c4d01f)

Sample files can also be found [here](https://drive.google.com/drive/folders/1JBWyRs-F5VOLYzDSadBknYmhoqAwRAOO?usp=sharing)

**Comment by cgsecurity** [comment]
Both files are already recovered by PhotoRec. It's not perfect as the end of the files with the non-standard extension is truncated.
Have you tried PhotoRec against a memory card from your Insta360 ?
On most action cams,  both high resolution and low resolution (lrv) videos are recorded in parallel after taking a still picture (insp for Insta360), so the files are fragmented preventing their recovery except by tools for the exact brand and model.

**Comment by amedrz** [comment]
> Have you tried PhotoRec against a memory card from your Insta360 ?

Yes I have tried to recover the files with my camera connected to my Mac and this is what the program shows

![IMG_1466](https://github.com/user-attachments/assets/c1d4c64e-8800-437a-843a-2a1985c9b3ea)

> so the files are fragmented preventing their recovery except by tools for the exact brand and model.
would this be the reason why they don't show up in the screenshot above?

**Comment by cgsecurity** [comment]
Can you play the mov files ? If the files are broken, it's probably due to the original file fragmentation.

**Comment by Anders-mobile** [comment]
@cgsecurity

I have a memory card that contains both deleted and non-deleted .insp and .lrv files. None of these files, whether deleted or still present, show up during the recovery process. As I understand, files that have not been deleted and are not corrupted should normally be visible during recovery.

The fact that none of these file types appear, regardless of their status, leads me to believe that there may be another issue at play here. Could this be related to how these file formats are fragmented or structured on the memory card? Or is there something specific in how PhotoRec handles these file formats that I should be aware of?

Thank you for any insights!
